packages/block-library/src/quote/transforms.js

```diff
@@ -127,15 +141,7 @@ const transforms = {
 				),
 		},
 	],
-	ungroup: ( { citation }, innerBlocks ) =>
-		citation
-			? [
-					...innerBlocks,
-					createBlock( 'core/paragraph', {
-						content: citation,
-					} ),
-			  ]
-			: innerBlocks,
+	ungroup,
```

Comment:
What might be better here is introduce a new key "unwrap" and only show these at the top of the block switcher. Then this also wouldn't show up as "Ungroup" in the Options dropdown